docs/notebooks/installation.rst

```diff
@@ -1,77 +1,167 @@
 
 .. code-block:: bash
 
    pip install neptune-notebooks
 
-Then enable extension for your Jupyter:
+2. Enable the extension for Jupyter:
 
 .. code-block:: bash
 
    jupyter nbextension enable --py neptune-notebooks
 
-Remember to install Neptune-client, if you did not do so already:
+3. Install Neptune client, if you did not do so already:
 
 .. code-block:: bash
 
    pip install neptune-client
 
-JupyterLab
-----------
 
-Requirements
-^^^^^^^^^^^^
-`Node.js <https://nodejs.org/en>`_ and `npm <https://www.npmjs.com/get-npm>`_ are required.
+For JupyterLab
+---------------
 
-To check if you have Node.js installed, run this command in your terminal:
+JupyterLab is the next-generation web-based UI for Project Jupyter. 
 
-.. code-block:: bash
+When you install JupyterLab using ``pip install jupyterlab``, the standard Jupyter is also installed, as a dependency.
+
+
+Installation methods
+^^^^^^^^^^^^^^^^^^^^
+You can install the extension using any of the following three methods:
+
+- Through the command line (full neptune-notebooks package)
+- Through the command line (JupyterLab extension only)
+- Using the Extension Manager (JupyterLab extension only)
+
+**Requirements**
+
+Irrespective of which method you use, the following must be preinstalled on your system before you begin.
+
+- `Node.js <https://nodejs.org/en>`_ 
+- `npm <https://www.npmjs.com/get-npm>`_
+- `JupyterLab <https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html>`_
+
+.. tip:: When installing Python packages, it is best practice to work in a `virtual environment <https://virtualenv.pypa.io/en/latest/>`_. 
+
+
+Method 1: Install the full neptune-notebooks package through the command line
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Pros**
+
+This is the most comprehensive type of installation:
+
+- It includes extensions for both standard Jupyter and JupyterLab. 
+- It also installs a CLI for uploading Notebook files.
+- The the neptune-client package is a dependency.
+
+**Cons**
 
-    node -v
+- After the installation, you still have to perform some manual actions in JupyterLab. 
+- This method is not the preferred way of installing extensions in JupyterLab.
 
+**Procedure**
 
-To confirm that you have npm installed you can run this command in your terminal:
+1. In the console, run:
 
 .. code-block:: bash
 
-    npm -v
+    pip install neptune-notebooks
 
-If you do not have them, please refer to: `installation instructions <https://www.npmjs.com/get-npm>`_.
+2. Start JupyterLab by running:
 
-Using command line
-^^^^^^^^^^^^^^^^^^
-Install `neptune-notebooks <https://www.npmjs.com/package/neptune-notebooks>`_ for your JupyterLab. In terminal run:
+.. code-block:: bash
+
+    jupyter lab
+
+3. In the "Build Recommended" notification that is displayed, click **Build**.
+4. In the "Build Complete" notification that is displayed, click **Reload**.
```

Comment:
It takes some time before the build process is completed and the last notification is displayed (minutes, even) - maybe we can mention that?

Reply:
I have added a note:
so:
3. In the "Build Recommended" notification that is displayed, click **Build**.

.. note:: The build process takes up to several minutes.